### PR TITLE
Add config for sqlite "soft_heap_limit" 

### PR DESCRIPTION
**Pull request by JuneKelly**
We can set a limit to the size of the sqlite heap, via config, which may help with the memory problems we've been having .

We should set `GIT_BRIDGE_SQLITE_HEAP_LIMIT_BYTES` to `512000000` in prod.

### CODE REVIEW

File: src/main/java/uk/ac/ic/wlgitbridge/bridge/db/sqlite/update/alter/SetSoftHeapLimitPragma.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+public class SetSoftHeapLimitPragma implements SQLUpdate {
+    private int heapLimitBytes = 0;
+
+    public SetSoftHeapLimitPragma(int heapLimitBytes) {
+        this.heapLimitBytes = heapLimitBytes;
+    }
+
+    @Override
+    public String getSQL() {
+        return "PRAGMA soft_heap_limit="+this.heapLimitBytes+";";
```
Comment (JuneKelly) [inline src/main/java/uk/ac/ic/wlgitbridge/bridge/db/sqlite/update/alter/SetSoftHeapLimitPragma.java:14]: It's not ideal to build a query this way, but prepared statements don't seem to work with pragmas and parameters, so....

Comment (gh2k) [reply src/main/java/uk/ac/ic/wlgitbridge/bridge/db/sqlite/update/alter/SetSoftHeapLimitPragma.java:14]: Agreed... - We know it's an integer, so it's safe enough.

**Review by gh2k — APPROVED** [review]
LGTM - Thanks!

Can we sqash when merging please? :-)

**Comment by JuneKelly** [comment]
Squashed

**Pull request merged**